inject-java/src/test/groovy/io/micronaut/annotation/processing/visitor/JavaReconstructionSpec.groovy

```diff
@@ -423,8 +423,10 @@ class Test<T> {
 }
 """)
         def fieldType = classElement.fields[0].type
+        List<? extends GenericPlaceholderElement> placeholders = fieldType.getDeclaredGenericPlaceholders()
 
         expect:
+        assert placeholders.every { it.nativeType.class.simpleName == "TypeVar" }
```

Comment:
assert is not necessary here

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/7081/commits/34c232a5b4bc9f909aaa37fde496e4bf10a68c63